### PR TITLE
Updated to latest version of eventuate-messaging-kafka. Used thread class-loader. https://github.com/eventuate-tram/eventuate…

### PR DESCRIPTION
**Pull request by dartartem**
…-tram-sagas/issues/30

### CODE REVIEW

File: eventuate-tram-spring-consumer-kafka/src/main/java/io/eventuate/tram/spring/consumer/kafka/EventuateTramKafkaMessageConsumerConfiguration.java [hunk 1]
```diff
@@ -11,7 +12,7 @@
 import org.springframework.context.annotation.Import;
 
 @Configuration
-@Import({TramConsumerCommonConfiguration.class, MessageConsumerKafkaConfiguration.class})
+@Import({TramConsumerCommonConfiguration.class, MessageConsumerKafkaConfiguration.class, KafkaConsumerFactoryConfiguration.class})
```
Comment (cer) [inline eventuate-tram-spring-consumer-kafka/src/main/java/io/eventuate/tram/spring/consumer/kafka/EventuateTramKafkaMessageConsumerConfiguration.java:15]: What's the purpose of this change? It seems unrelated to classloading?

Comment (dialtahi) [reply eventuate-tram-spring-consumer-kafka/src/main/java/io/eventuate/tram/spring/consumer/kafka/EventuateTramKafkaMessageConsumerConfiguration.java:15]: This change is related to this one in order to inject the kafka consumer factory:
https://github.com/eventuate-tram/eventuate-tram-core/pull/106/files

Comment (dartartem) [reply eventuate-tram-spring-consumer-kafka/src/main/java/io/eventuate/tram/spring/consumer/kafka/EventuateTramKafkaMessageConsumerConfiguration.java:15]: Chris, now kafka messaging project contains this change: https://github.com/eventuate-foundation/eventuate-messaging-kafka/pull/35/files
It makes necessary to have KafkaConsumerFactory to created consumers in eventuate local and tram.
And when I tested changes I also updated eventuate-messaging-kafka project with snapshot of eventuate-common
to avoid any conflicts between versions of eventuate-common in eventuate local, eventuate tram, eventuate tram sagas.
Anyway we need to update all libraries step by step and this change is required.
Please let me remind you how dependencies look like:
eventuate-common->eventuate-common-messaging->eventuate-messaging-kafka

eventuate-common->eventuate-local
eventuate-messaging-kafka->eventuate-local

eventuate-common->eventuate-tram-core
eventuate-messaging-kafka->eventuate-tram-core

eventuate-common->eventuate-tram-sagas
eventuate-messaging-kafka->eventuate-tram-sagas

Comment (dartartem) [reply eventuate-tram-spring-consumer-kafka/src/main/java/io/eventuate/tram/spring/consumer/kafka/EventuateTramKafkaMessageConsumerConfiguration.java:15]: @dialtahi thank you, probably you are right, I will review dependencies, there could be also issues with micronaut.
